src/auto/keymap_key.rs

```diff
@@ -0,0 +1,5 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use gdk_sys;
```

Comment:
Considering that this file is mostly empty, no need to generate it.

Reply:
Oups, i missed this one. Fixed